cmake/cmake_celix/templates/MANIFEST.json.in

```diff
@@ -0,0 +1,14 @@
+{
+    $<JOIN:$<TARGET_PROPERTY:${BUNDLE_TARGET_NAME},BUNDLE_HEADERS>,$<COMMA>
+    >$<$<BOOL:$<TARGET_PROPERTY:${BUNDLE_TARGET_NAME},BUNDLE_HEADERS>>:$<COMMA>>
+    "CELIX_BUNDLE_MANIFEST_VERSION" : "2.0.0",
+    "CELIX_BUNDLE_SYMBOLIC_NAME" : "$<TARGET_PROPERTY:${BUNDLE_TARGET_NAME},BUNDLE_SYMBOLIC_NAME>",
+    "CELIX_BUNDLE_VERSION" : "$<TARGET_PROPERTY:${BUNDLE_TARGET_NAME},BUNDLE_VERSION>",
```

Comment:
To make strict typing work, this need to be a version string (`celix_properties_isVersionString`).